src/brpc/rtmp.cpp

```diff
@@ -63,15 +63,19 @@ FlvWriter::FlvWriter(butil::IOBuf* buf)
     : _write_header(false), _buf(buf) {
 }
 
-static char g_flv_header[9] = { 'F', 'L', 'V', 0x01, 0x05, 0, 0, 0, 0x09 };
+FlvWriter::FlvWriter(butil::IOBuf* buf, const FlvWriterOptions& options)
+    : _write_header(false), _buf(buf) {
+    const int flags_bit_index = 4;
+    _header[flags_bit_index] = static_cast<uint8_t>(options.flv_content_type);
+}
 
 butil::Status FlvWriter::Write(const RtmpVideoMessage& msg) {
```

Comment:
char header[9] = { 'F', 'L', 'V', 0x01, _options.flv_content_type, 0, 0, 0, 0x09 };

Reply:
Thanks for your feedback. It prompts that "cannot convert that enum type to char in initialization", so I do a type casting in d8003ebe34ecc73bb4d6806b455078afd1cd5b81. Please let me know if you have any concerns.